.github/workflows/build-rock.yaml

```
@@ -13,35 +13,27 @@ on:
         default: "latest/stable"
         type: string
       source-branch:
-        description: Github branch to checkout. If blank, it will use default values.
-        default: '' # For default behaviour, see https://github.com/actions/checkout/tree/v4/#usage
+        description: "Github branch to checkout. If blank, it will use default values."
+        default: ''
         required: false
         type: string
+
     outputs:
-        rock-artifact:
-          description: "Name of the artifact that the built ROCK has been uploaded to."
-          value: ${{ jobs.build-rock.outputs.rock-artifact}}
-        rock-reference:
-          description: "Rock reference to be used in order to copy the image to docker cache.
-            It consists of <rock-name>:<rock-version>."
-          value: ${{ jobs.build-rock.outputs.rock-reference}}
-        rock-filename:
-          description: "Filename of the .rock file produced"
-          value: ${{ jobs.build-rock.outputs.rock-filename}}
+      rock-artifact:
+        description: "Name of the artifact that the built ROCK has been uploaded to."
+        value: ${{ jobs.build-rock.outputs.rock-artifact }}
+      rock-filename:
+        description: "Filename of the .rock file produced"
+        value: ${{ jobs.build-rock.outputs.rock-filename }}
 
 jobs:
   build-rock:
     name: Build ${{ inputs.rock-dir }}
     runs-on: ubuntu-22.04
```

nitpick: Update image
```suggestion
    runs-on: ubuntu-24.04
```